include/unk_020507CC.h

```diff
@@ -5,12 +5,12 @@
 #include "savedata.h"
 
 int FieldEvents_SaveSize(void);
-void FieldEvents_Init(UnkStruct_020507E4 * param0);
-UnkStruct_020507E4 * SaveData_Events(SaveData * param0);
-BOOL sub_020507F0(UnkStruct_020507E4 * param0, u16 param1);
-void sub_0205081C(UnkStruct_020507E4 * param0, u16 param1);
-void sub_02050844(UnkStruct_020507E4 * param0, u16 param1);
-u8 * sub_02050870(UnkStruct_020507E4 * param0, u16 param1);
-u16 * sub_020508B8(UnkStruct_020507E4 * param0, u16 param1);
+void FieldEvents_Init(FieldEvents * param0);
```

Comment:
Is this, e.g., data from `res/field/events`?

Reply:
I'll need to double check but I believe so

Reply:
Ok this function specifically just 0 inits it lmao

Reply:
I more meant the concept, specifically. Like, what is a `FieldEvent`, contextually?

To me, looking at `sub_02050870`, these look like the save data flags and variables. Can we check it against HG?

Reply:
I'm not sure, I named it after the already named function SaveData_GetEvents, so whoever named that would know